parity-instantseal/instant-seal-config.toml

```diff
@@ -26,7 +28,7 @@ reserved_only = true
 #  JSON-RPC will be listening for connections on IP 0.0.0.0.
 interface = "all"
 # Allows Cross-Origin Requests from domain '*'.
-cors = "*"
+cors = ["*"]
```

Comment:
Did the syntax change in 1.9.7, or was this wrong before?

Reply:
https://github.com/paritytech/parity/pull/7387